lms/validation/_launch_params.py

```diff
@@ -64,7 +65,9 @@ def lti_user(self):
         except HTTPUnprocessableEntity as err:
             raise ValidationError(err.json) from err
 
-        return LTIUser(kwargs["user_id"], kwargs["oauth_consumer_key"])
+        return LTIUser(
+            kwargs["user_id"], kwargs["oauth_consumer_key"], kwargs.get("roles", "")
+        )
```

Comment:
Adding `roles` to `LaunchParamsSchema` ensures that when LTI launch params are deserialized into an `LTIUser` the LTI roles param is deserialized to `LTIUser.roles`.